src/UnitTests/AddProfiles.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Shouldly;
+using Xunit;
+
+namespace AutoMapper.UnitTests
+{
+    public class AddProfiles : SpecBase
+    {
+        public class Source { }
+        public class Dest { }
+
+        public class ForwardProfile : Profile
+        {
+            public ForwardProfile()
+            {
+                CreateMap<Source, Dest>();
+            }
+        }
+
+        public class ReverseProfile : Profile
+        {
+            public ReverseProfile()
+            {
+                CreateMap<Dest, Source>();
+            }
+        }
+
+        [Fact]
+        public void Should_not_throw_when_loading_multiple_profiles()
+        {
+            IEnumerable<Profile> profiles = new Profile[] { new ForwardProfile(), new ReverseProfile() };
+            var config = new MapperConfiguration(cfg => cfg.AddProfiles(profiles));
+
+            config.AssertConfigurationIsValid();
```

Comment:
This tests only asserts the method exists, it doesn't really check it adds both profiles.

Reply:
@jpridx the `MapperConfiguration` object has a `Profile` property that you can assert against. Configuration validation can succeed if there are zero maps.

Reply:
I added a test that verifies that the profiles were added. @jbogard thank you for pointing me to the Profiles property.